powerstore/resource_filesystem_snapshot.go

```diff
@@ -84,9 +92,6 @@ func (r *resourceFileSystemSnapshot) Schema(ctx context.Context, req resource.Sc
 				Computed:            true,
 				Description:         "Description of the filesystem snapshot.",
 				MarkdownDescription: "Description of the filesystem snapshot.",
-				Validators: []validator.String{
-					stringvalidator.LengthAtLeast(1),
-				},
 			},
 			"expiration_timestamp": schema.StringAttribute{
```

Comment:
For expiration_timestamp, though regex is applied, it is still accepting "". is it valid value?

Reply:
the expirtation timestamp "" is valid value to set infinite retention